book/src/bridge/extern_rustqt.md

```diff
@@ -30,6 +30,13 @@ A bridge module may contain zero or more `extern "RustQt"` blocks.
 This complements the [`extern "Rust"` CXX section](https://cxx.rs/extern-rust.html)
 but allows for declaring Qt specific features on C++ types.
 
+## Automatic case conversion
```

Comment:
this is not specific to the `RustQt` block and should work on the `C++Qt` block, maybe merge this text with the `attributes` text and link to that header from somewhere in here?